.gitignore

```diff
@@ -107,3 +107,5 @@ venv.bak/
 # mypy
 .mypy_cache/
 .DS_Store
+
+.idea
```

Comment:
Seems you can remove this if the `.idea` is created by you?

Reply:
removed